generators/client/templates/angular/src/main/webapp/app/shared/_shared-libs.module.ts

```diff
@@ -5,6 +5,9 @@ import { CommonModule } from '@angular/common';
 import { NgbModule } from '@ng-bootstrap/ng-bootstrap';
 import { NgJhipsterModule } from 'ng-jhipster';
 import { InfiniteScrollModule } from 'angular2-infinite-scroll';
+<%_ if (websocket === 'spring-websocket') { _%>
+import { WindowRef } from './tracker/window.service';
```

Comment:
can you move this to shared-common.module as this file should only have external libs. same goes for provider def below